packages/story-editor/src/components/panels/design/textStyle/fontPicker.js

```diff
@@ -121,6 +119,7 @@ const FontPicker = forwardRef(function FontPicker(
     return map;
   }, [fonts]);
 
+  // TODO: Add support for custom fonts as well.
```

Comment:
Thanks for pushing the PR in WIP state, I can take a more thorough look in a few hours. Just to confirm -- this TODO of adding the custom fonts to the Dropdown here is intended for #9880, right? I was planning to look into this today (unless you already secretly did it yourself).

Reply:
This TODO is for enqueueing font stylesheets for custom fonts when displaying them in the dropdown.

The custom fonts themselves are already visible and searchable in the dropdown.

Reply:
What‘s left to be done in that other ticket is adding a dedicated section for the custom fonts though.

Reply:
Yep, that dedicated section was what I had in mind for looking into :)